src/BenchmarkDotNet.Core/Extensions/CommonExtensions.cs

```diff
@@ -60,6 +61,30 @@ public static string ToStr(this double value, string format = "0.##")
             return string.Format(HostEnvironmentInfo.MainCultureInfo, $"{{0:{format}}}", args);
         }
 
+        /// <summary>
+        /// Gets column title formatted using the specified style
+        /// </summary>
+        public static string GetColumnTitle(this IColumn column, ISummaryStyle style)
+        {
+            if (!style.PrintUnitsInHeader)
+            {
+                return column.ColumnName;
+            }
+
+            if (column.UnitType == UnitType.Size)
```

Comment:
Sure. I was just thinking how F#'s pattern matching would guarantee that all cases are handled, even when someone adds a new enum member.